src/typescript-service.ts

```diff
@@ -429,6 +429,7 @@ export class TypeScriptService {
 		const symbols = iterate(configs)
 			.map(config => this._collectWorkspaceSymbols(config, query || symbolQuery, limit))
 			.flatten<SymbolInformation>()
+			.filter(symbol => !symbol.location.uri.includes('/node_modules/'))
```

Comment:
Sorry, not following? You mean if you pass a symbol query that is defined in a dependency?